controllers/apartments.rb

```diff
@@ -1,14 +1,57 @@
+get "/" do # Gets the path "/" index
+  @all_apartments = "apartments/index"
+  @search_apartment = "apartments/search"
+  @show_apartment = "apartments/show"
+  @add_apartment = "apartments/new"
+  @list_tenants = "/apartments/3/tenants"
+  @add_tenant = "/apartments/3/tenants/new"
+  erb :"welcome" # Finds index.erb template
+end
+
 # index
-get '/apartments' do
-  erb :"apartments/index"
+get "/apartments" do # This is the index route for apartments
+  @header_route = "Apartment List"
```

Comment:
This content can live hard-coded in the `index.erb` file, because it never changes for that view.